src/Wrapper.js

```diff
@@ -154,36 +142,23 @@ export default (Component) => {
       }
     }
 
-    hasValue() {
-      return this.state.value !== '';
-    }
+    hasValue = () => this.state.value !== '';
 
-    isFormDisabled() {
-      return this.context.formsy.isFormDisabled();
-    }
+    isFormDisabled = () => this.context.formsy.isFormDisabled();
 
-    isFormSubmitted() {
-      return this.state.formSubmitted;
-    }
+    isFormSubmitted = () => this.state.formSubmitted;
 
-    isPristine() {
-      return this.state.isPristine;
-    }
+    isPristine = () => this.state.isPristine;
 
-    isRequired() {
-      return !!this.props.required;
-    }
+    isRequired = () => !!this.props.required;
 
-    isValid() {
-      return this.state.isValid;
-    }
+    isValid = () => this.state.isValid;
 
-    isValidValue(value) {
-      return this.context.formsy.isValidValue.call(null, this, value);
+    isValidValue = value =>
+      this.context.formsy.isValidValue.call(null, this, value);
       // return this.props.isValidValue.call(null, this, value);
```

Comment:
I may need to do a sweep of the codebase for commented out code...

Reply:
Shall I clean them up in a separate PR?

Reply:
Would you mind removing this commented out line? I don't see any reason we would keep it around.

Reply:
@zabute I actually searched around and couldn't find any others imminently. If you see more, then yeah, we can just leave them for a future PR.